src/NexusMods.App.UI/LeftMenu/Items/LaunchButtonViewModel.cs

```diff
@@ -20,26 +22,26 @@ public class LaunchButtonViewModel : AViewModel<ILaunchButtonViewModel>, ILaunch
 
     [Reactive] public Percent? Progress { get; set; }
 
-    private readonly ILoadoutRegistry _loadoutRegistry;
     private readonly IToolManager _toolManager;
+    private readonly IConnection _conn;
 
     public LaunchButtonViewModel(ILogger<LaunchButtonViewModel> logger, IToolManager toolManager,
-        IActivityMonitor manager, ILoadoutRegistry loadoutRegistry)
+        IActivityMonitor manager, IConnection conn)
     {
         _toolManager = toolManager;
-        _loadoutRegistry = loadoutRegistry;
+        _conn = conn;
 
         Command = ReactiveCommand.CreateFromObservable(() => Observable.StartAsync(LaunchGame, RxApp.TaskpoolScheduler));
     }
 
     private async Task LaunchGame(CancellationToken token)
     {
         Label = Language.LaunchButtonViewModel_LaunchGame_RUNNING;
-        var marker = _loadoutRegistry.GetMarker(LoadoutId);
-        var tool = _toolManager.GetTools(marker.Value).OfType<IRunGameTool>().First();
+        var marker = _conn.Db.Get(LoadoutId);
```

Comment:
I saw a bunch of these `_conn.Db.Get(Id)` all throughout the code.
Should something like the VM of a UI button really interact directly with the database?
Shouldn't the implementation of this be deferred to something like the `LoadoutRegistry`?